src/ompl/geometric/planners/bitstar/src/BITstar.cpp

```diff
@@ -934,9 +950,10 @@ namespace ompl
 
 
             //And then do the for starts. We do this last as the starts are added to the queue, which uses a cost-to-go heuristic in it's ordering, and for that we want all the goals updated.
-            //If newStart is NULL, then no new start exists. The following line both assigns newStart and checks if it is a valid pointer.
-            while (static_cast<bool>((newStart = Planner::pis_.nextStart())) == true)
+            while (pis_.haveMoreStartStates())
```

Comment:
I used the same horrible while condition for consistency between the goal and start loops. :wink: